omf-python/src/file/reader.rs

```diff
@@ -136,4 +137,23 @@ impl PyReader {
             .collect::<Result<Vec<_>, _>>()
             .map_err(|e| PyErr::new::<PyValueError, _>(e.to_string()))
     }
+
+    pub fn array_image<'p>(
```

Comment:
I guess this is a case where we're not using the existing API because we don't want to unnecessarily decode the image, which probably makes sense for how we will use it. Maybe this should have a name like image_bytes() - I note the similar function image() drops array I guess because it returns a single image not an array of images.

Reply:
Done